app/src/sharedTest/java/org/oppia/android/app/home/promotedlist/PromotedStoryViewModelTest.kt

```diff
@@ -242,25 +189,24 @@ class PromotedStoryViewModelTest {
           /* activity = */ it,
           /* internalProfileId = */1,
           /* totalStoryCount = */3,
-          /* entytType = */"",
+          /* entytType = */"entity",
```

Comment:
Please update these cases to also use named arguments, consistently spaced variable/comments, and/or helpers as needed to simplify the tests. Ditto below.

Reply:
Ah, thanks for reiterating this as well!
Used named arguments but it seems silly to create a helper that just repeats the ViewModel constructor (as opposed to the createBasicViewModel() helper that sets the view model to a basic case). Let me know if there's simpler solution

Reply:
You could also create a helper that has default arguments and override those on a case-by-case basis, but I think what you have now seems good.